lib/recurly/unique_coupon_list.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class Recurly_UniqueCouponList extends Recurly_Pager
+{
+  public static function get($params = null, $client = null)
+  {
```

Comment:
I *think* the coding standard this follows is to just do the new line before `{ ` on the class. For the functions it seems to be on the same line. So lets join this onto the previous line.